<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ApurvaNaik42" failures="1" tests="3" name="Error Validation Test" time="17.867" errors="0" timestamp="2023-02-03T00:20:25 IST">
  <testcase classname="org.example.ErrormessageScenariosTest" name="errorMsgValidation" time="0.384"/>
  <testcase classname="org.example.ErrormessageScenariosTest" name="productCartvalidation" time="3.658">
    <skipped/>
  </testcase> <!-- productCartvalidation -->
  <testcase classname="org.example.ErrormessageScenariosTest" name="productCartvalidation" time="3.602">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.example.ErrormessageScenariosTest.productCartvalidation(ErrormessageScenariosTest.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- productCartvalidation -->
</testsuite> <!-- Error Validation Test -->
